src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -91,28 +94,31 @@ const ActiveState = ({
                         return (
                             <ActivityItem
                                 key={item.type + item.id}
-                                className="bcs-activity-feed-comment"
-                                data-testid="comment"
+                                className={hasReplies ? 'bcs-activity-thread' : 'bcs-activity-feed-comment'}
```

Comment:
It doesn't look like `.bcs-activity-thread` is used at the moment. Will we need different styles for the ActivityItem component in the future?

Reply:
Probably we won't need to style this ActivityItem differently. I was just following the pattern in the component, but noticed  'bcs-activity-feed-comment' is not used neither